<?xml version="1.0"?>
<launch>
  
  <node pkg="campusrover_costmap" type="campusrover_local_costmap" name="campusrover_local_costmap" output="screen">
    <param name="map_length" value="6.0"/>
    <param name="map_width" value="6.0"/>
    <param name="resolution" value="0.02"/>
    <param name="inflation_radius" value="0.05"/>
  </node>

</launch>
